feat(changelog): elastic-metal-added-proxmox-backup-server-4-available 2026-07-07 (#6667)
scaleway/docs-content: changelog/july2026/2026-07-07-elastic-metal-added-proxmox-backup-server-4-available.mdx

---
title: Proxmox Backup Server 4 available
status: added
date: 2026-07-07
category: bare-metal
product: elastic-metal
---

> We have now released version 4 of Proxmox Backup Server based on Debian 13 (Trixie) across all Beryllium, Iridium and Lithium offers, and in several different Availability Zones.

Reviewer: ```suggestion
Proxmox Backup Server version 4, based on Debian 13 (Trixie), is now available across all Beryllium, Iridium, and Lithium offers, in multiple Availability Zones.
```

Reviewer 2: @firdevs-a, for the changelog I think maybe the original author doesn't notice when we add a suggestion (I'm not sure actually). Given that they use a specific tool for writing the changelog, there may not be a way for feedback to go back to them. I'm just saying because it looks like this changelog has been stuck here for a while. I think you can apply the suggestion, and if there is no reason to wait for publication, we can probably merge this PR.
